legal-api/migrations/versions/b9f84f7a7c44_add_login_source_to_users.py

```diff
@@ -0,0 +1,25 @@
+"""add_login_source_to_users
+
+Revision ID: b9f84f7a7c44
+Revises: 1ba027b074a5
+Create Date: 2022-11-24 11:23:57.467443
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'b9f84f7a7c44'
+down_revision = '1ba027b074a5'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('users', sa.Column('login_source', sa.String(length=200)))
```

Comment:
need to add same column to `users_version` table too.  

I had forgotten about the `users_version` table when you added the `idp_userid` column.  We need to add `idp_userid` to it too.

Reply:
Done.